pkgs/development/python-modules/cyclopts/default.nix

```diff
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "cyclopts";
-  version = "3.2.0";
+  version = "3.2.1";
```

Comment:
Looks like this was added by accident? But since it's just a minor bump. I will just test as it is.

Reply:
Looks like it didn't even cause any rebuilds. Maybe this was just a missing rebase.

Reply:
Argh, I was trying to fixup the first commit, I must have accidentally rewritten the previous commit as well.